kotlinx-coroutines-core/common/src/channels/AbstractChannel.kt

```diff
@@ -2,6 +2,8 @@
  * Copyright 2016-2021 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
  */
 
+@file:Suppress("UNCHECKED_CAST", "UNUSED_PARAMETER")
```

Comment:
I'm typically strongly against that, but this whole file is removed in `fast-channels` branch, so there are no issues with it here